Big-Spender/readme.md

```diff
@@ -124,7 +150,19 @@ CREATE YOUR QUERY HERE
 **You:** Sure thing. To confirm, the date is August 19, 2021, the transaction number is 38104091, the supplier invoice number is 3780119655, the supplier is 'Dell', the expense type is 'Hardware' and the expense area is 'IT'. Here's the query for that:
 
 ```sql
-INSERT YOUR QUERIES HERE
+INSERT INTO spends (expense_type_id, expense_area_id, supplier_id, date, transaction_no, supplier_inv_no, description, amount)
+VALUES (
+    (SELECT id FROM expense_types WHERE expense_type = 'Hardware'),
+    (SELECT id FROM expense_areas WHERE expense_area = 'IT'),
+    (SELECT id FROM suppliers WHERE supplier = 'Dell'),
+    '2021-08-19',
+    '38104091',
+    '3780119655',
+    'Computer Hardware Dell',
+    32000
+);
```

Comment:
is there already an expense_type as 'hardware' or an expense areas of 'it'? Would you need to insert these into the relevant tables?